test/params_api_test.c

```diff
@@ -69,7 +69,7 @@ static int test_param_type_extra(OSSL_PARAM *param, const unsigned char *cmp,
     const int sizet = bit32 && sizeof(size_t) > sizeof(int32_t);
     const int signd = param->data_type == OSSL_PARAM_INTEGER;
 
-    OSSL_PARAM_set_unmodified(param);
+    OSSL_PARAM_set_all_unmodified(param);
```

Comment:
The flip side is that `param` now *must* be a properly terminated OSSL_PARAM array.  Because `test_param_int()` makes a singular OSSL_PARAM, this call barfs all over Travis for the moment...

Reply:
You could revert this back to `param->return_size = OSSL_PARAM_UNMODIFIED`...

Reply:
I've done this with a comment and added a test that uses the API call properly on an array.